dist/ExtUtils-CBuilder/lib/ExtUtils/CBuilder/Platform/msys.pm

```diff
@@ -0,0 +1,33 @@
+package ExtUtils::CBuilder::Platform::msys;
```

Comment:
If this is exactly the same as `ExtUtils::CBuilder::Platform::cygwin`, shouldn't it just inherit from that instead?